presto-benchto-benchmarks/src/test/java/com/facebook/presto/sql/planner/AbstractCostBasedPlanTest.java

```diff
@@ -78,7 +78,7 @@ public void test(String queryResourcePath)
 
     private String getQueryPlanResourcePath(String queryResourcePath)
     {
-        return queryResourcePath.replaceAll("\\.sql$", ".plan.txt");
+        return queryResourcePath.replaceAll("\\.sql$", ".plan");
```

Comment:
Is this rename strictly necessary ?

Reply:
I put it there just to appease the CI header check. For whatever reason we have a CI license header check in the presto repo that uses a script from the velox repo. So the only way to get the PR check to pass is to modify the script in the velox repo, change the github check, or modify the filenames so the velox header check doesn't detect it.

I chose the latter, but can revert that commit. I did it at least temporarily to make the CI happy.

Reply:
>we have a CI license header check in the presto repo that uses a script from the velox repo.
Let's make an issue for this

Reply:
I've filed #21875